nautobot/docs/administration/nautobot-server.md

```diff
@@ -151,12 +151,14 @@ nautobot=> \q
     - `extras.job` should now be included in the dump (removed `--exclude extras.job` from the example usage)
     - `django_rq` should now be excluded from the dump (added `--exclude django_rq` to the example usage)
 
++/- 1.5.22
+    - We do not recommend at this time using `--natural-primary` as this can result in inconsistent or incorrect data for data models that use GenericForeignKeys, such as `Cable`, `Note`, `ObjectChange`, and `Tag`.
+
 ```no-highlight
 nautobot-server dumpdata \
   --natural-foreign \
-  --natural-primary \
```

Comment:
Are there any instances where exporting with `natural-foreign` but without `natural-primary` might result in import issues?

Reply:
If we remove `natural-foreign` will we have to add `ContentType` to export too?

Reply:
If a plugin implements model tests inheriting from `nautobot.core.testing.models.ModelTestCases.BaseModelTestCase` then it does include testing of the natural and composite keys for the model. This is a relatively new addition though and not all models even in core have tests of this sort yet.

Reply:
> If we remove `natural-foreign` will we have to add `ContentType` to export too?

I think so, yes. Which will also mean we need to document explicitly deleting all existing ContentTypes before importing.

Reply:
> > If we remove `natural-foreign` will we have to add `ContentType` to export too?
> 
> I think so, yes. Which will also mean we need to document explicitly deleting all existing ContentTypes before importing.

I think this would be preferred if we intend on encouraging (or at least just denoting that it is the primary use case) dump/loaddata being used only for wholesale migration and not merging datasets. We could call out that using `natural-foreign` can be used to make other migrations easier, but not 100% battle-tested.

If I'm not mistaken, the only risk for importing `ContentType` into an empty DB is if an app with model(s) is missing or new ones are added, but both should be cleaned up in `post_upgrade`.